extensions/resteasy/deployment/src/main/java/io/quarkus/resteasy/deployment/ResteasyStandaloneBuildStep.java

```diff
@@ -56,7 +56,7 @@ public ResteasyStandaloneBuildItem(String deploymentRootPath) {
 
     }
 
-    @BuildStep()
+    @BuildStep(providesCapabilities = Capabilities.RESTEASY)
```

Comment:
Is there any specific reason that both this one and the `providesCapabilities` on `build` are needed?

Reply:
In fact no, I've annotated both as both the build step are supposed to provide a resteasy's `FeatureBuildItem` so I've replicated the behavior to the related `BuildStep` annotation.

My first tough was to add a `BuildProducer<CapabilityBuildItem>` to both the methods and conditionally add the capability but then I noticed that one of the two builder step are also leveraging capabilities so adding a build producer could trigger a cycle.

An option would be to add an additional BuildItem to signal that resteasy is in use and add the capability.

Reply:
We probably should get rid of 'providesCapabilities' and just use a normal build item for it, the reason why it is treated specially is largely for historical reasons.